tox.ini

```diff
@@ -1,43 +1,24 @@
 [tox]
-envlist = py38-django{32, 42}
+envlist = py{38, 312}-django{42}
 
 [doc8]
 ignore = D000, D001
 
 [pycodestyle]
 exclude = .git,.tox,migrations
-; default ignores: E121,E123,E126,E133,E226,E241,E242,E704,W503,W504,W505
-; E501 (line too long) duplicates pylint check
 ignore = E121,E123,E126,E133,E226,E241,E242,E704,W503,W504,W505,E501
 
 [pydocstyle]
-; D101 = Missing docstring in public class
-; D200 = One-line docstring should fit on one line with quotes
-; D203 = 1 blank line required before class docstring
-; D212 = Multi-line docstring summary should start at the first line
-; D215 = Section underline is over-indented (numpy style)
-; D404 = First word of the docstring should not be This (numpy style)
-; D405 = Section name should be properly capitalized (numpy style)
-; D406 = Section name should end with a newline (numpy style)
-; D407 = Missing dashed underline after section (numpy style)
-; D408 = Section underline should be in the line following the section’s name (numpy style)
-; D409 = Section underline should match the length of its name (numpy style)
-; D410 = Missing blank line after section (numpy style)
-; D411 = Missing blank line before section (numpy style)
-; D412 = No blank lines allowed between a section header and its content (numpy style)
-; D413 = Missing blank line after last section (numpy style)
-; D414 = Section has no content (numpy style)
```

Comment:
It would be nice not to drop these comments.

Reply:
Our default config parser drops these comments but we make sure to keep these when reviewing the changes.

Reply:
Something you may want to consider, so someone doesn't forget: https://github.com/edx/edx-internal/blob/master/.github/workflows/protect-comments.yml